examples/get_started/json-csv-reader.py

```diff
@@ -88,16 +87,7 @@ def main():
     print("========================================================================")
     static_csv_ds = DataChain.from_csv(uri, output=ChatDialog, object_name="chat")
     static_csv_ds.print_schema()
-    print(static_csv_ds.to_pandas())
-
-    uri = "gs://datachain-demo/laion-aesthetics-csv"
-    print()
-    print("========================================================================")
-    print("dynamic CSV with header schema test parsing 3/3M objects")
-    print("========================================================================")
-    dynamic_csv_ds = DataChain.from_csv(uri, object_name="laion", nrows=3)
```

Comment:
[F] `nrows` with a `csv` from `gs` does not work on my machine (never returns). We could leave the example in with `nrows=3` removed but we will get a blow out in run time. I am happy to go with whatever is decided.